docs/source/hpxml_to_openstudio.rst

```diff
@@ -159,6 +159,9 @@ The local shelter coefficient can be entered as ``Site/extension/ShelterCoeffici
 The shelter coefficient is defined by the AIM-2 infiltration model to account for nearby buildings, trees and obstructions.
 If not provided, the value of 0.5 will be assumed.
 
+Whether there is a flue or chimney (associated with the heating system or water heater) can be optionally specified with ``HasFlueOrChimney``.
+If not provided, it is assumed that there is no flue or chimney.
```

Comment:
I'm on the fence here. It's okay to assume false if not provided, but we could also consider doing smarter defaulting of this input (based on the type of heating/DHW systems, presence of fireplace, etc.). @jmaguire1, any thoughts?

Reply:
For some WH's (condensing gas comes to mind), you generally don't have a central flue but have a short run of PVC out the wall. Generally that's only possible with condensing WHs since they exhaust at a lower temperature (because of the condensation). I'd imagine that's true of HVAC as well, although I'm not as confident on that. So we could definitely do some smart defaulting based on the equipment type and the presence of a fireplace, but it might be more complicated than just if they're fuel fired or not.

That being said, I definitely do think we could default this smartly, I could give you the efficiency levels where a gas WH would require a central flue. We might want to tap someone else on the HVAC side (maybe Jon) to check what we ought to assume there.

Reply:
I found some old guidance from Eric Wilson:

- Any furnace with < 89% AFUE
- Any boiler with < 89% AFUE
- Any water heater with EF < 0.63 (double check with Jeff)
- Gas fireplace

Reply:
That sounds pretty reasonable to me. For water heaters that'd be power vent, condensing, and tankless, all of which probably wouldn't be natural draft.

Reply:
@shorowit @jmaguire1 Is this for non-electric only?

Reply:
Yes, non-electric only. Though there shouldn't be any electric HVAC/WHs with efficiencies this low.